# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/mp3info/usr-local.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./Makefile~	Fri Aug 18 17:42:22 2000
+++ ./Makefile	Sat Jan 13 21:51:56 2001
@@ -28,7 +28,7 @@
 # bindir = where binaries get installed (default = /usr/local/bin)
 # mandir = where the manual page gets installed (default = /usr/local/man/man1)
 
-prefix=/usr/local
+prefix=/usr
 bindir=${prefix}/bin
 mandir = $(prefix)/man/man1
 
